public/js/monitor.js

```diff
@@ -213,6 +213,14 @@ function styleActiveLink(locationHref) {
 ( async() => {
   document.addEventListener("touchstart", function(){}, true);
   const win = window;
+  if (win.navigator.language.startsWith("de")) {
```

Comment:
I _think_ this is OK. I tried looking around [Pontoon](https://pontoon.mozilla.org/teams/) to see if there were any other locales that may start with "de" but didn't see anything.

But https://www.andiamo.co.uk/resources/iso-language-codes/ seems to imply that all of these may be valid ISO 639-1 standard language codes that may potentially match the `/^de/` style regexp (and all are German, so I think we're safe here):


Language (Region) | Code
-- | --
German (Austria) | de-at
German (Liechtenstein) | de-li
German (Luxembourg) | de-lu
German (Standard) | de
German (Switzerland) | de-ch

Reply:
We only support one German (`de`), no local variants.